content/docs/command-reference/cache/dir.md

```diff
@@ -25,21 +25,30 @@ Unlike doing so with `dvc config cache`, `dvc cache dir` transform paths
 absolute path, then it's preserved as it is.
 
 If no path `value` is provided to this command, it prints the path for current
-cache directory.
+cache directory. The value is searched in the system, global, project, and local
+config files by default (in that order). Options `--system`, `--global`,
+`--project`, and `--local` can be used to look only in one place.
+
+The `value` is written to the project's config file by default (`.dvc/config`).
+Options `--system`, `--global`, and `--local` can be used to write to the other
+possible config files.
 
 ## Options
 
-- `--global` - modify a global config file (e.g. `~/.config/dvc/config`) instead
-  of the project's `.dvc/config`.
+- `--global` - use a global config file (e.g. `~/.config/dvc/config`) instead of
```

Comment:
Please try to reuse the option texts from my previous review here and in all other places if possible.